plugins/ltac/tauto.ml

```diff
@@ -189,31 +189,34 @@ let flatten_contravariant_disj _ ist =
       tclTHEN (tclTHENLIST tacs) tac0
   | _ -> fail
 
+let evalglobref_of_globref =
+  function
+  | GlobRef.VarRef v -> EvalVarRef v
+  | GlobRef.ConstRef c -> EvalConstRef c
+  | GlobRef.IndRef _ | GlobRef.ConstructRef _ -> assert false
+
 let make_unfold name =
-  let dir = DirPath.make (List.map Id.of_string ["Logic"; "Init"; "Coq"]) in
-  let const = Constant.make2 (ModPath.MPfile dir) (Label.make name) in
-  Locus.(AllOccurrences, ArgArg (EvalConstRef const, None))
+  let const = evalglobref_of_globref (Coqlib.lib_ref name) in
+  Locus.(AllOccurrences, ArgArg (const, None))
 
-let u_not = make_unfold "not"
+let u_not = lazy (make_unfold "core.not.type")
```

Comment:
Shouldn't you make this a thunk rather than a delayed value? What if we rebind the reference mid-flight? Other uses of `lib_ref` seem to prefer this pattern.

Reply:
Done. Thanks.